auth.py

```diff
@@ -1,15 +1,21 @@
 """This is the module that contains functions related to authenticating to GitHub with a personal access token."""
 
+import logging
+import logging.config
+
 import github3
 import requests
 
+logging.basicConfig(level=logging.DEBUG)
+
```

Comment:
What information is going to this log? I don't see anywhere that directly references the function i'd expect: [debug()](https://docs.python.org/3/library/logging.html#logging.Logger.debug), [info()](https://docs.python.org/3/library/logging.html#logging.Logger.info), [warning()](https://docs.python.org/3/library/logging.html#logging.Logger.warning), [error()](https://docs.python.org/3/library/logging.html#logging.Logger.error) and [critical()](https://docs.python.org/3/library/logging.html#logging.Logger.critical) methods

Reply:
> What information is going to this log? I don't see anywhere that directly references the function i'd expect: [debug()](https://docs.python.org/3/library/logging.html#logging.Logger.debug), [info()](https://docs.python.org/3/library/logging.html#logging.Logger.info), [warning()](https://docs.python.org/3/library/logging.html#logging.Logger.warning), [error()](https://docs.python.org/3/library/logging.html#logging.Logger.error) and [critical()](https://docs.python.org/3/library/logging.html#logging.Logger.critical) methods

🤦 

I believe @ricardojdsilva87 meant to remove this.  They did at one point and it snuck back in.  It was temporary for them.  @ricardojdsilva87 is that correct?  Could you please do a quick follow PR to remove if so?

Reply:
[This was the removal commit](9972b96763715d422ab56ee9354966ecdf2c48db)

Reply:
Nm.  This was a specific commit.  Final boss version it is not there.  We are good.  My bad.

Reply:
Thanks for checking my understanding on this. I'll make sure my review tab is looking at all commits next time.